packages/ckeditor5-code-block/src/outdentcodeblockcommand.ts

```diff
@@ -31,13 +34,13 @@ export default class OutdentCodeBlockCommand extends Command {
 		 * @private
 		 * @member {String}
 		 */
-		this._indentSequence = editor.config.get( 'codeBlock.indentSequence' );
+		this._indentSequence = editor.config.get( 'codeBlock.indentSequence' ) as string;
```

Comment:
```suggestion
		this._indentSequence = editor.config.get( 'codeBlock.indentSequence' )!;
```

Same as in the indent command above. 

Docs should be moved to property declaration.